source/developers/sts-for-operator.rst

```diff
@@ -0,0 +1,132 @@
+.. _minio-sts-operator:
+
+===============================================
+Security Token Service (STS) for MinIO Operator
+===============================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. versionadded:: Operator v5.0.0
+
+   The MinIO Operator supports a set of API calls that allows an application to obtain STS credentials for a MinIO Tenant.
+
+:ref:`STS credentials <minio-security-token-service>` allow an application to access objects on a MinIO Tenant without the need to create any credentials for the application on the tenant.
```

Comment:
`any` could maybe be dropped here, it doesn't feel like it's adding much.

Reply:
Agreed